analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Verifier.cs

```diff
@@ -191,10 +282,10 @@ static IEnumerable<TMessage> ReadProtobuf(string path)
             }
         }
 
-        public static void VerifyUtilityAnalyzerIsNotRun(IEnumerable<string> paths,
-                                                           UtilityAnalyzerBase diagnosticAnalyzer,
-                                                           string protobufPath,
-                                                           IEnumerable<ParseOptions> options = null)
+        public static void VerifyNonConcurrentUtilityAnalyzerIsNotRun(IEnumerable<string> paths,
```

Comment:
This one doesn't need a rename, because there's no concurrent overload. Let's just add a short documentation comment.